src/interfaces.ts

```diff
@@ -181,7 +181,8 @@ export interface JobHelpers extends Helpers {
   ): Promise<QueryResult<R>>;
 
   /**
-   * An AbortSignal that will be triggered when the job should exit.
+   * A Node `AbortSignal` that will be triggered when the job should exit. It is
```

Comment:
You say a Node one, but AbortSignal is a DOM standard that Node implements I think? Not sure this needs specifying since there’s no ambiguity?

Reply:
This is a good point. I think I first added the word Node before I added the extra bit of clarity to the dm. I think it's fine without the word "Node"